extensions/amp-fit-text/validator-amp-fit-text.protoascii

```diff
@@ -14,10 +14,21 @@
 # limitations under the license.
 #
 
-tags: {  # amp-fit-text
+tags: {  # amp-fit-text in websites
   html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-fit-text"
+    version: "0.1"
+    version: "1.0"
+    version: "latest"
```

Comment:
Why does this change remove those specifications? I don't see them previously here, are they default values?

Reply:
Hey @twifkak, recently updated this PR to follow the guidance described in cl/366320502 and #33478, can you please (1) do one more code review pass of these changes, and (2) verify that this will not cause errors on existing pages?

Reply:
Ping @twifkak, PTAL